src/dataprotection/azext_dataprotection/manual/_help.py

```diff
@@ -55,6 +55,14 @@
         text: az dataprotection backup-instance list-from-resourcegraph --resource-groups resourceGroup --vaults vault --protection-status ProtectionError --datasource-type AzureDisk
 """
 
+helps['dataprotection backup-instance update-msi-permissions'] = """
+    type: command
+    short-summary: Assign missing roles to the vault based on the backup instance or restore request object.
```

Comment:
Apart from assigning roles to vault, some other things are also being done. Check with Aditya once if this help text still makes sense.

Reply:
Also remove restore request object from the help text for now.